docs/attributes-registry/http.md

```diff
@@ -1,32 +1,30 @@
+| ------------------------------ | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `http.connection.state`        | string   | State of the HTTP connection in the HTTP connection pool.                                                                                                                                                                                                                                                                    | `active`                                                                                                              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `http.request.body.size`       | int      | The size of the request payload body in bytes. This is the number of bytes transferred excluding headers and is often, but not always, present as the [Content-Length](https://www.rfc-editor.org/rfc/rfc9110.html#field.content-length) header. For requests using transport encoding, this should be the compressed size.  | `3495`                                                                                                                | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `http.request.header.<key>`    | string[] | HTTP request headers, `<key>` being the normalized HTTP Header name (lowercase), the value being the header values. [1]                                                                                                                                                                                                      | `http.request.header.content-type=["application/json"]`; `http.request.header.x-forwarded-for=["1.2.3.4", "1.2.3.5"]` | ![Stable](https://img.shields.io/badge/-stable-lightgreen)       |
+| `http.request.method`          | string   | HTTP request method. [2]                                                                                                                                                                                                                                                                                                     | `CONNECT`                                                                                                             | ![Stable](https://img.shields.io/badge/-stable-lightgreen)       |
```

Comment:
The examples changed here.. now it's just `CONNECT`. Any idea why?

Reply:
Fixed.